Packs/Claroty/Playbooks/playbook-Claroty_Incident.yml

```diff
@@ -0,0 +1,175 @@
+id: 54e6d2f8-8f27-4291-8ab7-e81480780c0e
+inputs: []
+name: Claroty Incident
+fromversion: "5.0.0"
+outputs: []
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "1"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      description: Manually resolved problem.
+      id: 8e712abe-51e2-4176-842f-47e507439fce
+      iscommand: false
+      name: ""
+      version: -1
+    taskid: 8e712abe-51e2-4176-842f-47e507439fce
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+  "1":
+    id: "1"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "3"
+    note: false
+    scriptarguments:
+      alert_number: {}
+      date_from: {}
+      fields:
+        simple: all
+      get_single_alert:
+        simple: ${incident.clarotyresourceid}
+      sort_by: {}
+      type: {}
+    separatecontext: false
+    task:
+      brand: ""
+      description: Gets alerts from CTD.
+      id: f93c6971-01b9-46f3-8ac9-ad481971cd02
+      iscommand: true
+      name: Claroty Get Alert Details
+      script: '|||claroty-query-alerts'
+      type: regular
+      version: -1
+    taskid: f93c6971-01b9-46f3-8ac9-ad481971cd02
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+  "3":
+    id: "3"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      description: Manually resolved problem.
+      id: b52451f8-1d4a-4af4-89d2-04939791cd9b
+      iscommand: false
+      name: Manual Resolution
+      type: regular
+      version: -1
+    taskid: b52451f8-1d4a-4af4-89d2-04939791cd9b
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 370
+        }
+      }
+  "4":
+    id: "4"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "5"
+    note: false
+    scriptarguments:
+      excluded_alerts: {}
+      resolve_as:
+        simple: resolve
+      select_all:
```

Comment:
i think we should remove this one

Reply:
@fvigo Help me out here - I removed the option to select all alerts to resolve, but I need to send False in the request data. What am I supposed to do here?

Reply:
@asaf96 you removed it in the integration, Itay's comment is that you left it in the playbook.